src/connectors/refinement-list/__tests__/connectRefinementList-test.js

```diff
@@ -457,4 +458,305 @@ describe('connectRefinementList', () => {
 
     expect(rendering.lastCall.args[0].hasExhaustiveItems).toEqual(false);
   });
+
+  it('can search in facet values', () => {
+    const widget = makeWidget({
+      attributeName: 'category',
+      limit: 2,
+    });
+
+    const helper = algoliasearchHelper(
+      fakeClient,
+      '',
+      widget.getConfiguration({})
+    );
+    helper.search = sinon.stub();
```

Comment:
Maybe prefer `jest.fn`. We should avoid to add new code with `sinon`. We probably remove it at some point.